src/spyglass/decoding/v1/sorted_spikes.py

```diff
@@ -417,48 +371,12 @@ def load_spike_data(key, filter_by_interval=True):
 
         return new_spike_times
 
-    @classmethod
-    def get_spike_indicator(cls, key, time):
-        time = np.asarray(time)
-        min_time, max_time = time[[0, -1]]
-        spike_times = cls.load_spike_data(key)
-        spike_indicator = np.zeros((len(time), len(spike_times)))
-
-        for ind, times in enumerate(spike_times):
-            times = times[np.logical_and(times >= min_time, times <= max_time)]
-            spike_indicator[:, ind] = np.bincount(
-                np.digitize(times, time[1:-1]),
-                minlength=time.shape[0],
-            )
-
-        return spike_indicator
-
-    @classmethod
-    def get_firing_rate(cls, key, time, multiunit=False):
-        spike_indicator = cls.get_spike_indicator(key, time)
-        if spike_indicator.ndim == 1:
-            spike_indicator = spike_indicator[:, np.newaxis]
-
-        sampling_frequency = 1 / np.median(np.diff(time))
-
-        if multiunit:
-            spike_indicator = spike_indicator.sum(axis=1, keepdims=True)
-        return np.stack(
-            [
-                get_multiunit_population_firing_rate(
-                    indicator[:, np.newaxis], sampling_frequency
-                )
-                for indicator in spike_indicator.T
-            ],
-            axis=1,
-        )
-
     def spike_times_sorted_by_place_field_peak(self, time_slice=None):
         if time_slice is None:
             time_slice = slice(-np.inf, np.inf)
 
-        spike_times = self.load_spike_data(self.proj())
-        classifier = self.load_model()
+        spike_times = self.fetch_spike_data(self.proj())
```

Comment:
```suggestion
        spike_times = self.fetch_spike_data(self.fetch1())
```
At least in my testing, `fetch_spike_data` required a dictionary key rather than a table restriction